array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,18 @@
+/* exported tail */
+
+// dec func
+// delete the first index in the array 
+// return the array arg
+
+function tail(array){
+  delete array[0]; 
+  var arrayTail = [];
+  for (var i = 0; i < array.length; i++){
+    if( i === 0 ){
```

Comment:
Instead of having a conditional run every iteration of this loop, you should just start the loop one index further along.